File: DevOps+/AWS+/CDK+/JavaCdkProject/src/main/java/app/stack/s3/bucket/S3BucketStack.java
package app.stack.s3.bucket;

import software.amazon.awscdk.Stack;
import software.amazon.awscdk.services.s3.Bucket;
import software.constructs.Construct;

import static software.amazon.awscdk.RemovalPolicy.DESTROY;

public class S3BucketStack extends Stack {
    public S3BucketStack(final Construct scope) {
        super(scope, S3BucketStack.class.getSimpleName(), null);
        Bucket.Builder.create(this, "Bucket resource creation")
                .bucketName("my-cdk-bucket-ei9a9l10slx")
                .removalPolicy(DESTROY)
                .build();
    }
}

File: DevOps+/AWS+/CDK+/PythonCdkProject/src/python_cdk_project/s3_bucket_stack.py
from aws_cdk import Stack, RemovalPolicy
from aws_cdk.aws_s3 import Bucket
from constructs import Construct


class S3BucketStack(Stack):

    def __init__(self, scope: Construct, construct_id: str, **kwargs) -> None:
        super().__init__(scope, construct_id, **kwargs)

        bucket_name: str = "cdk-bucket-14888403871340275063719"
        Bucket(
            self,
            id=bucket_name,
            bucket_name=bucket_name,
            removal_policy=RemovalPolicy.DESTROY,
            auto_delete_objects=True
        )
